package.json

```diff
@@ -36,6 +36,70 @@
     ],
     "main": "./main.js",
     "contributes": {
+        "chatParticipants": [
+            {
+                "id": "ms-azuretools.azure-agent",
```

Comment:
Can the ID be different?

Reply:
> Can the ID be different?

I think that would be a good thing to do from a telemetry standpoint. Should make sure there's nothing weird experience wise from that though.

Reply:
Testing suggests it doesn't really work well if that's done.